src/assets/custom/css/_sass/_services-page-intro.scss

```diff
@@ -1,23 +1,143 @@
   }
-  @include large {
-    height: 0;
-    padding-top: 650 / 1820 * 100%;
-  }
+
   @include extra-large {
-    @include max-width--framed;
-    height: 465px;
+    height: $header-background-nav-image-height--large-and-extra-large;
+    margin-bottom: $vertical-overlap;
+  }
+}
+
+.services-page-intro__navigation {
+  @include max-width--out-there;
+  align-items: center;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+
+  @include small {
+    padding: 5% 0;
+  }
+
+  @include medium {
+    height: $header-background-nav-image-height--medium;
+  }
+
+  @include large-and-extra-large {
+    height: $header-background-nav-height--large-and-extra-large;
+    padding-left: $header-background-nav-image-padding--large-and-extra-large;
+    padding-right: $header-background-nav-image-padding--large-and-extra-large;
+  }
+}
+
+.services-page-intro__icon-container {
+  @include small {
+    align-items: center;
+    display: flex;
+    flex-basis: 50%;
+    flex-direction: column;
+    justify-content: center;
+    padding: 5% 5%;
+  }
+}
+
+$icon-image-height--large-extra-large: 150px;
+$icon-image-height--small-medium: 115px;
+$icon-image-width--large-extra-large: 130px;
+$icon-image-width--small-medium: 100px;
+
+.services-page-intro__navigation-link {
+  color: white;
+
+  & .service-line-icon {
+    height: 55%;
+    fill: white;
+  }
+
+  &:hover, &:focus {
+    text-decoration: none;
+    transition: color 0.3s ease;
+    color: $grey--light;
+
+    .service-line-icon {
+      transition: fill 0.3s ease;
+    }
+  }
+
+  @include small-and-medium {
+    width: $icon-image-width--small-medium;
+  }
+
+  @include large-and-extra-large {
+    width: $icon-image-width--large-extra-large;
+  }
+}
+
+.services-page-intro__icon--software-modernisation {
+  background-image: url("#{$site-base-url}/assets/custom/img/services/software-modernisation-hexagon.png");
+}
+
+.services-page-intro__icon--software-delivery {
+  background-image: url("#{$site-base-url}/assets/custom/img/services/software-delivery-hexagon.png");
+}
+
+.services-page-intro__icon--strategic-advice {
+  background-image: url("#{$site-base-url}/assets/custom/img/services/strategic-advice-hexagon.png");
+}
+
+.services-page-intro__icon--specialist-expertise {
+  background-image: url("#{$site-base-url}/assets/custom/img/services/specialist-expertise-hexagon.png");
+}
+
+.services-page-intro__navigation-icon {
+  align-items: center;
+  background-size: cover;
+  display: flex;
+  justify-content: center;
+
+  @include small {
+    width: $icon-image-width--small-medium;
+  }
+
+  @include small-and-medium {
+    height: $icon-image-height--small-medium
+  }
+
+  @include large-and-extra-large {
+    height: $icon-image-height--large-extra-large;
+  }
+}
+
+.services-page-intro__navigation-text {
+  @include bold;
+  text-align: center;
+  
+  @include small-and-medium {
+    @include base;
```

Comment:
I think we have a `base-bold` and a `mercer-bold` for line 137. This may be more descriptive and specific.